lib/index.js

```diff
@@ -129,7 +151,7 @@ class Modem {
     const ascending = options.sort === 'ascending' ? 1 : defaultOptions.ascending
     const unreadPreferred = options.unreadPreferred === true ? 1 : defaultOptions.unreadPreferred
 
-    const response = await this.apiRequest('/sms/sms-list', {
+    const response = await this.apiRequest(...versionedApiRequest(this.webUiVersion, '/api/sms/sms-list', {
```

Comment:
The `path` of this request starts with `/api/` unlike the others. Is this a mistake or is this how it works on your version? This would create a request to `/api/api/sms/sms-list`, right?